lib/prompts/selectPublicAppPrompt.ts

```diff
@@ -1,25 +1,31 @@
-// @ts-nocheck
-const { promptUser } = require('./promptUtils');
-const { i18n } = require('../lang');
-const { uiLine } = require('../ui');
-const { logError } = require('../errorHandlers/index');
-const { logger } = require('@hubspot/local-dev-lib/logger');
-const {
-  fetchPublicAppsForPortal,
-} = require('@hubspot/local-dev-lib/api/appsDev');
-const { EXIT_CODES } = require('../enums/exitCodes');
+import { promptUser } from './promptUtils';
+import { i18n } from '../lang';
+import { uiLine } from '../ui';
+import { logError } from '../errorHandlers/index';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { fetchPublicAppsForPortal } from '@hubspot/local-dev-lib/api/appsDev';
+import { EXIT_CODES } from '../enums/exitCodes';
+import { PublicApp } from '@hubspot/local-dev-lib/types/Apps';
 
 const i18nKey = 'lib.prompts.selectPublicAppPrompt';
 
-const fetchPublicAppOptions = async (
-  accountId,
-  accountName,
+type PublicAppPromptResponse = {
+  appId: number;
+};
+
+async function fetchPublicAppOptions(
+  accountId: number | null,
+  accountName: string,
   isMigratingApp = false
-) => {
+): Promise<PublicApp[]> {
   try {
-    const {
-      data: { results: publicApps },
-    } = await fetchPublicAppsForPortal(accountId);
+    let publicApps: PublicApp[] = [];
```

Comment:
Similar thing here to the comment on your other PR - this slightly changes behavior if the accountId doesn't exist. Before, an error would be thrown by `fetchPublicAppsForPortal`, but now it will log the errors that show up when you don't have any public apps, and exit with a success code

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1293/commits/64ec64e3b160db98474cf96fff85513b85b11095